conf/server.properties

```diff
@@ -0,0 +1,33 @@
+# FullyQualifiedServerUrl is used only in the case of offline content generation
```

Comment:
Is this file used in any way to build the Pentaho container image? If not, please remove it from git.

Reply:
Removed in git and we will pass this in k8s